master/static/srv/notebook-entrypoint.sh

```diff
@@ -18,6 +18,31 @@ set -x
 test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}"
 set +x
 
+if [ "$PACHD_ADDRESS" != "" ] && [ "$SKIP_PACHYDERM_INSTALL" != "true" ]; then
```

Comment:
in hindsight, i kinda wish this was all keyed off `det_intergrations_pachd=true` rather than the ip. rip.